examples/exampleRealTimeReporter/tests/home.js

```diff
@@ -22,13 +22,13 @@ describe(suiteName, function() {
   });
 
   after((browser, done) => {
-    PublicReportingAPI.finishSuite();
+    PublicReportingAPI.finishSuite({ name: suiteName });
```

Comment:
Will this function have any other arguments apart of name? Is it neccessery to build a new object instead of passing name directly?

Reply:
No, just suite name.
Fixed